core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -478,6 +478,7 @@ trait TestDriverComponent extends DriverComponent with DataAccess with DefaultIn
       .subscriptionId(UUID.randomUUID())
       .cloudPlatform(bio.terra.profile.model.CloudPlatform.AZURE)
       .managedResourceGroupId("fake-mrg")
+      .createdDate("2023-09-12T22:20:48.949Z")
```

Comment:
Because this is set to 9/12, the existing clone tests verify that "new" billing projects can be used.